README.md

```diff
@@ -339,8 +339,61 @@ LIT_SERVER_API_KEY=supersecretkey python main.py
 Clients are expected to auth with the same API key set in the `X-API-Key` HTTP header.
 
 </details>
+
+
+<details>
+  <summary>Stream long responses</summary>
+
 &nbsp;
 
+`LitServer` can stream longer responses, such as LLM generated text.
+
+To enable streaming, you need to implement `LitAPI.predict` and `LitAPI.encode_response` as a generator (a Python
+function that yields output) and set `LitServer(..., stream=True)`.
+
+For example, streaming long responses generated over time:
+
+```python
+from typing import Generator
+
+from litserve.api import LitAPI
+import json
+
+from litserve import LitServer
+from pydantic import BaseModel
+
+
+class Request(BaseModel):
+    number: int
+
+
+class SimpleStreamAPI(LitAPI):
+    def setup(self, device) -> None:
+        self.num_outputs = 100
+
+    def decode_request(self, request: Request) -> str:
+        return request.number
+
```

Comment:
same thing here... you're introducing and documenting STREAMING not the pydantic API.